server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationSent.ts

```diff
@@ -13,23 +14,28 @@ export default class SufficientInformationSent implements TasklistPage {
 
   title = 'How to get further information'
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-function
-  constructor(public readonly body: Record<string, unknown>) {}
+  userName: Body['name']
+
+  emailAddress: Body['emailAddress']
 
-  user: ApprovedPremisesUser
+  phoneNumber: Body['phoneNumber']
 
-  static async initialize(
-    body: Record<string, unknown>,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  constructor(
+    public readonly body: Record<string, unknown>,
     assessment: Assessment,
-    token: string,
-    dataServices: DataServices,
   ) {
-    const user = await dataServices.userService.getUserById(token, assessment.application.createdByUserId)
-
-    const page = new SufficientInformationSent(body)
-    page.user = user
-
-    return page
+    this.userName = retrieveOptionalQuestionResponseFromFormArtifact(assessment.application, ConfirmYourDetails, 'name')
```

Comment:
Please could we provide a fall back here in case `retrieveOptionalQuestionResponseFromFormArtifact` returns `undefined`?